packages/instant/src/containers/connected_account_payment_method.ts

```diff
@@ -49,7 +55,24 @@ const mergeProps = (
     onInstallWalletClick: connectedDispatch.onInstallWalletClick,
     walletName: connectedState.providerState.name,
     onUnlockWalletClick: () => {
-        connectedDispatch.unlockWalletAndDispatchToStore(connectedState.providerState);
+        const isMobile = envUtil.isMobileOperatingSystem();
+        if (!isMobile) {
+            connectedDispatch.unlockWalletAndDispatchToStore(connectedState.providerState);
+            return;
+        }
+        const operatingSystem = envUtil.getOperatingSystem();
+        let url = COINBASE_WALLET_SITE_URL;
```

Comment:
it looks like we're doing this type of logic in 2 places -- can we introduce a helper ?

Reply:
This should be fixed now